### PR TITLE
Article: Create NPM Module [in progress]

### PR DESCRIPTION
**Pull request by jakegardner**
closes https://github.com/FreeCodeCamp/wiki/issues/812

### CODE REVIEW

**Comment by alayek** [comment]
@jakegardner where is your file?

**Comment by abhisekp** [comment]
@theflametrooper @alayek Please read https://github.com/FreeCodeCamp/wiki/issues/812#issuecomment-217756871

**Comment by jakegardner** [comment]
First draft done, will review and squash tomorrow. Feel free to comment.

**Comment by abhisekp** [comment]
@jakegardner let us review it and squash only right before merging. :+1:

File: Create-NPM-Module.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+####Step 2: Create package information
+
+The next step is to create the information to publish with your package, stored in `package.json`.
+NPM provides a wizard to make creating this file easy.
+
+Simply run `npm init` and answer the questions.
+
+Your `package.json` should look something like this:
+
+```
```
Comment (theflametrooper) [inline Create-NPM-Module.md]: Use `json` syntax highlighting.

File: Create-NPM-Module.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
+##Create an NPM Module
```
Comment (theflametrooper) [inline Create-NPM-Module.md]: ~~##Create~~ ## Create

File: Create-NPM-Module.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+An NPM module is a set of Javascript functionality bundled into a distributable package.
+[NPM](http://www.npmjs.com) maintains the registry of all available packages, and is also the tool used to install
+packages from its registry. 
+
+The beauty of NPM is that you can assemble packages other people have created to create something new,
+and someone later on could use the package you create. If you have some code that you want to share with the world, 
+publishing a module to NPM is easy.
+
+####Step 1: Create your script
```
Comment (theflametrooper) [inline Create-NPM-Module.md]: Same as [above](https://github.com/FreeCodeCamp/wiki/pull/884/files#r62623452). Add spacing between you're `##` and the actual title.

File: Create-NPM-Module.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+I created this simple utility as `index.js`:
+
+```
+var time = new Date().toTimeString();
+console.log(time);
+```
+
+####Step 2: Create package information
```
Comment (theflametrooper) [inline Create-NPM-Module.md]: Same as [above](https://github.com/FreeCodeCamp/wiki/pull/884/files#r62623452). Add spacing between you're `##` and the actual title.

File: Create-NPM-Module.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+The beauty of NPM is that you can assemble packages other people have created to create something new,
+and someone later on could use the package you create. If you have some code that you want to share with the world, 
+publishing a module to NPM is easy.
+
+####Step 1: Create your script
+
+I created this simple utility as `index.js`:
+
+```
+var time = new Date().toTimeString();
```
Comment (theflametrooper) [inline Create-NPM-Module.md]: Use `js` syntax highlighting.

File: Create-NPM-Module.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+  "description": "accurate time retrieval",
+  "main": "index.js",
+  "author": "HoursAndMinutes",
+  "license": "ISC"
+}
+```
+
+Don't forget to include the GitHub repository link if the code is on GitHub!
+
+####Step 3: Create a user account on NPM
```
Comment (theflametrooper) [inline Create-NPM-Module.md]: Same as [above](https://github.com/FreeCodeCamp/wiki/pull/884/files#r62623452). Add spacing between you're `##` and the actual title.

File: Create-NPM-Module.md [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+
+In order to publish a package to NPM, you need to have an account registered. To do that, run `npm adduser`.
+You can verify the account currently in use with `npm config ls`.
+
+####Step 4: Publish to NPM
```
Comment (theflametrooper) [inline Create-NPM-Module.md]: Same as [above](https://github.com/FreeCodeCamp/wiki/pull/884/files#r62623452). Add spacing between you're `##` and the actual title.

File: Create-NPM-Module.md (hunk 4 above)
Comment (Rafase282) [inline Create-NPM-Module.md]:
```diff
-####Step 2: Create package information
+#### Step 2: Create package information
```

**Comment by jakegardner** [comment]
Anything else? Any feedback on content?

File: Create-NPM-Module.md [hunk 8]
```diff
@@ -0,0 +1,51 @@
+## Create an NPM Module
```
Comment (Rafase282) [inline Create-NPM-Module.md]:
```diff
-## Create an NPM Module
+# Create an NPM MOdule
```

File: Create-NPM-Module.md [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+
+An NPM module is a set of Javascript functionality bundled into a distributable package.
+[NPM](http://www.npmjs.com) maintains the registry of all available packages, and is also the tool used to install
+packages from its registry. 
+
+The beauty of NPM is that you can assemble packages other people have created to create something new,
+and someone later on could use the package you create. If you have some code that you want to share with the world, 
+publishing a module to NPM is easy.
+
+#### Step 1: Create your script
```
Comment (Rafase282) [inline Create-NPM-Module.md]:
```diff
-#### Step 1: Create your script
+## Step 1: Create your script
```

File: Create-NPM-Module.md [hunk 10]
```diff
@@ -0,0 +1,51 @@
...
+
+I created this simple utility as `index.js`:
+
+```javascript
+var time = new Date().toTimeString();
+console.log(time);
+```
+
+#### Step 2: Create package information
```
Comment (Rafase282) [inline Create-NPM-Module.md]:
```diff
-#### Step 2: Create package information
+## Step 2: Create package information
```

File: Create-NPM-Module.md [hunk 11]
```diff
@@ -0,0 +1,51 @@
...
+  "description": "accurate time retrieval",
+  "main": "index.js",
+  "author": "HoursAndMinutes",
+  "license": "ISC"
+}
+```
+
+Don't forget to include the GitHub repository link if the code is on GitHub!
+
+#### Step 3: Create a user account on NPM
```
Comment (Rafase282) [inline Create-NPM-Module.md]:
```diff
-#### Step 3: Create a user account on NPM
+## Step 3: Create a user account on NPM
```

File: Create-NPM-Module.md [hunk 12]
```diff
@@ -0,0 +1,51 @@
...
+
+In order to publish a package to NPM, you need to have an account registered. To do that, run `npm adduser`.
+You can verify the account currently in use with `npm config ls`.
+
+#### Step 4: Publish to NPM
```
Comment (Rafase282) [inline Create-NPM-Module.md]:
```diff
-#### Step 4: Publish to NPM
+## Step 4: Publish to NPM
```

**Comment by Rafase282** [comment]
The rest will be done when articles with C are done, thanks @jakegardner for your contribution.

**Pull request merged**